### DISCUSSION TITLE
Using tj-actions on Node16 after deprecation of Node12

### DISCUSSION THREAD

**Discussion by CadeSummers** [tj-actions/changed-files#1157] [Q&A] [answered]
As Github Actions deprecates node 12 and transitions to node 16, will there be updates for tj-actions for it to run utilizing node 16? Will the deprecation effect gha workflows utilizing tj-actions greatly?

See GitHub Blog: https://github.blog/changelog/2023-05-04-github-actions-all-actions-will-run-on-node16-instead-of-node12/

**Comment by jackton1** [comment, marked as answer]
@CadeSummers I believe we already use node16 so this wouldn't be an issue
